pyhms/demes/cma_deme.py

```diff
@@ -27,8 +82,16 @@ def __init__(
         if random_seed is not None:
             opts["randn"] = np.random.randn
             opts["seed"] = random_seed + self._started_at
+        if config.sigma0:
```

Comment:
I'm not sure if the presence of sigma0 should overwrite "set_stds" option. I would rather put checking for "CMA_stds" as the first condition in this if-elif-else statement. I understand, that "CMA_stds" are multipliers, so you expect None and then set the sigma0 to 1.0, but it may not be as clear at first sight and is probably unnecessarily limiting.